src/main/app/fact/FactApi.ts

```diff
@@ -113,6 +115,36 @@ export class FactApi {
       });
   }
 
+  public getContactTypes(): Promise<ContactType[]> {
+    return this.axios
+      .get(`${this.adminBaseUrl}/contactTypes`)
+      .then(results => results.data)
+      .catch(err => {
+        this.logError(err);
+        return Promise.reject(err);
+      });
+  }
```

Comment:
Just a quick question relating to Axios - do these promises need conditions introduced for timeouts? Utilising Promise.race()? This would be for all API requests so we can implement proper timeout handling, unless Axios already handled that and passes it as the error?

Reply:
Timeouts will be handled by the catch block. However, I think the timeout will need to be set - it appears to be zero at the moment. https://stackoverflow.com/questions/57995863/what-is-axios-default-timeout

Reply:
I tried setting a short timeout on the axios object in the constructor of this class, held the API call on a breakpoint so the call timed out and the error was caught in the catch block.